nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_listing-table.scss

```diff
@@ -25,7 +25,6 @@
         border-style: solid;
 
         table {
-            width: 100%;
```

Comment:
This change is leading to horizontal scrolling.

<img width="1595" alt="Screenshot 2024-03-25 at 1 01 26 PM" src="https://github.com/apache/nifi/assets/123395/e6970a44-babf-4c9e-8f89-3bd24c80c74d">

Reply:
ok I can restore this.